libraries/standard/json/src/json.c

```diff
@@ -366,7 +365,9 @@ static bool_ skipEscape( const char * buf,
     bool_ ret = false;
     size_t i = *start;
 
-    if( ( ( i + 1U ) < max ) && ( buf[ i ] == '\\' ) )
+    assert( buf[ i ] == '\\' );
```

Comment:
Moved the check to an assert @dan4thewin

Reply:
Why does this function require that the starting index be `\\`? From the documentation of the function, it wasn't obvious..

Reply:
Only makes sense to call skipEscape when the first character is an escape `"\\"`